wgpu-core/src/device/mod.rs

```diff
@@ -2767,7 +2767,9 @@ impl<G: GlobalIdentityHandlerFactory> Global<G> {
                         hal::memory::Dependencies::empty(),
                         iter::once(transition_src).chain(iter::once(transition_dst)),
                     );
-                    comb.copy_buffer(&stage_buffer, &buffer.raw, iter::once(region));
+                    if buffer.size > 0 {
```

Comment:
hmm, I'm surprised this is the only thing we had to do. What about `stage_buffer`? Was it created with size 0?

Reply:
https://github.com/gfx-rs/wgpu/pull/710 I think that this change ensures that stage_buffer is always created with at least size 1.